.github/workflows/publish.yml

```diff
@@ -32,6 +32,18 @@ jobs:
         with:
           tinygo-version: '0.27.0'
 
+      - name: Version 👍
+        id: version-bump
+        uses: jaywcjlove/github-action-package@v1.3.0
+        with:
+          version: ${{ github.event.inputs.release_version }}
+
+      - name: Install 🔧
+        run: npm install
+
+      - name: Build 🔧
+        run: npm run build
```

Comment:
Again doing the explicit build here in hopes of ensuring the cjs folder is included in what is published, also I moved the tag action to after the build so we don't add a tag if the action fails.